### DISCUSSION TITLE
Making "Collapsing sprite properties" addon more clear

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#6578] [Ideas]
I like a lot the "Collapsing sprite properties" addon, like any other addon that makes the relevant areas bigger. I think that, to make it more clear, the small black arrow should stay visible after having collapsed the properties area, so that you can reopen the area by clicking the arrow other than by double clicking the sprite list. I'm thinking of 80% of my students, that would click the arrow and than wouldn't know how to make the properties area visible again.

**Comment by WorldLanguages** [comment]
This, more or less?

![image](https://github.com/ScratchAddons/ScratchAddons/assets/17484114/ad7a976d-adc0-4acc-ace8-431d39e57896)

**Comment by sfederici** [comment]
Yes. If possible adding the black arrow without the white rectangle, so that it doesn't use space from the sprite list area.

**Comment by DNin01** [comment]
There is an `i` button which you can click to expand the properties.

**Reply by WorldLanguages** [reply]
Yeah, of course, but he's saying that most students probably wouldn't find out that easily, as they haven't enabled the addon themselves.

**Reply by sfederici** [reply]
Yes, exactly
